#ifndef __XARM_MSGS_H
#define __XARM_MSGS_H

#include <xarm_msgs/msg/robot_msg.hpp>
#include <xarm_msgs/msg/io_state.hpp>
#include <xarm_msgs/msg/cio_state.hpp>

#include <xarm_msgs/srv/bio_gripper_ctrl.hpp>
#include <xarm_msgs/srv/bio_gripper_enable.hpp>
#include <xarm_msgs/srv/call.hpp>
#include <xarm_msgs/srv/get_analog_io.hpp>
#include <xarm_msgs/srv/get_digital_io.hpp>
#include <xarm_msgs/srv/get_float32.hpp>
#include <xarm_msgs/srv/get_float32_list.hpp>
#include <xarm_msgs/srv/get_int16.hpp>
#include <xarm_msgs/srv/get_int16_list.hpp>
#include <xarm_msgs/srv/get_int32.hpp>
#include <xarm_msgs/srv/get_int32_by_type.hpp>
#include <xarm_msgs/srv/get_set_modbus_data.hpp>
#include <xarm_msgs/srv/gripper_move.hpp>
#include <xarm_msgs/srv/move_cartesian.hpp>
#include <xarm_msgs/srv/move_circle.hpp>
#include <xarm_msgs/srv/move_home.hpp>
#include <xarm_msgs/srv/move_joint.hpp>
#include <xarm_msgs/srv/move_velocity.hpp>
#include <xarm_msgs/srv/robotiq_activate.hpp>
#include <xarm_msgs/srv/robotiq_get_status.hpp>
#include <xarm_msgs/srv/robotiq_move.hpp>
#include <xarm_msgs/srv/robotiq_reset.hpp>
#include <xarm_msgs/srv/set_analog_io.hpp>
#include <xarm_msgs/srv/set_digital_io.hpp>
#include <xarm_msgs/srv/set_float32.hpp>
#include <xarm_msgs/srv/set_float32_list.hpp>
#include <xarm_msgs/srv/set_tcp_load.hpp>
#include <xarm_msgs/srv/set_int16.hpp>
#include <xarm_msgs/srv/set_int16_by_id.hpp>
#include <xarm_msgs/srv/set_int16_list.hpp>
#include <xarm_msgs/srv/set_int32.hpp>
#include <xarm_msgs/srv/set_int32_by_type.hpp>
#include <xarm_msgs/srv/traj_ctrl.hpp>
#include <xarm_msgs/srv/traj_play.hpp>
#include <xarm_msgs/srv/vacuum_gripper_ctrl.hpp>
#include <xarm_msgs/srv/set_modbus_timeout.hpp>

#endif // __XARM_MSGS_H